attributes/default.rb

```diff
@@ -20,6 +20,7 @@
 # version of telegraf to install, e.g. '0.10.0-1' or nil for the latest
 default['telegraf']['version'] = nil
 default['telegraf']['install_type'] = 'package'
+default['telegraf']['action'] = :create
```

Comment:
This is both unused and technically wrong (since you can't define a symbol in a policy file, I don't think the default should be something literally undefinable...) I'm going to remove this line. In the case of different actions ... it'll just do the action.